src/data/fin/vec_notation.lean

```diff
@@ -124,6 +124,13 @@ set.ext $ λ y, by simp [fin.exists_fin_succ, eq_comm]
 @[simp] lemma range_empty (u : fin 0 → α) : set.range u = ∅ :=
 set.range_eq_empty _
 
+@[simp] lemma range_fin_one (x : α) (u : fin 0 → α) : set.range (matrix.vec_cons x u) = {x} :=
+by rw [range_cons, range_empty, set.union_empty]
+
+@[simp] lemma range_fin_two (x y : α) (u : fin 0 → α) :
+  set.range (vec_cons x $ vec_cons y u) = {x, y} :=
+by rw [range_cons, range_fin_one, set.singleton_union]
```

Comment:
Sorry, I've changed my mind again; `range_fin_two` would be the statement that `set.range f = {f 0, f 1}`. So maybe `range_cons_empty` and `range_cons_cons_empty`?

Reply:
Yeah, that sounds better!